Resources/Prototypes/_ES/Entities/Objects/Weapons/Melee/sleepy_pen.yml

```
@@ -0,0 +1,28 @@
+- type: entity
+  name: pen
+  suffix: Sleepy Pen
+  parent: Pen # It is just like normal pen, isn't it?
+  description: A dark ink pen.
+  id: ESSleepyPen
```

```suggestion
  parent: Pen # It is just like normal pen, isn't it?
  id: ESSleepyPen
  name: pen
  description: A dark ink pen.
  suffix: ES, Sleepy Pen
```
insane ordering of fields here